lib/internal/fs/utils.js

```diff
@@ -781,7 +781,6 @@ const defaultRmOptions = {
 const defaultRmdirOptions = {
   retryDelay: 100,
   maxRetries: 0,
-  recursive: false,
 };
```

Comment:
This should be completely deleted with the corresponding validations. `retryDelay` and `maxRetries` are both tied to `recursive`.